core/src/test/java/com/graphhopper/reader/dem/MultiSourceElevationProviderTest.java

```diff
@@ -87,6 +87,6 @@ public void testGetEle() {
         assertEquals(108, instance.getEle(60.0000001, 16), precision);
         assertEquals(0, instance.getEle(60.0000001, 19), precision);
         // Stor Roten
-        assertEquals(14, instance.getEle(60.251, 18.805), precision);
+        assertEquals(4, instance.getEle(60.251, 18.805), precision);
```

Comment:
Can you move this change out into a separate PR? I think it is just the fix in core/MultiSourceElevationProvider.java which is unrelated to this PR?